carpool/views.py

```diff
@@ -47,12 +48,15 @@ def new(request):
             })
             to_email = form.cleaned_data.get('email')
             send_mail(mail_subject, message, 'Hackweek@example.com', [to_email], fail_silently=False)
-            done=True
+            HttpResponseRedirect("{% url 'polls.signupcomp' %}")
```

Comment:
Add a `return` before the function. Also, this URL syntax doesn't work. Use `HttpResponseRedirect("/signupcomp/")` instead.